code/controllers/subsystem/logging.dm

```diff
@@ -0,0 +1,42 @@
+SUBSYSTEM_DEF(logging)
+	name = "Logging"
+	flags = SS_NO_FIRE
+	var/list/entries = list()
+
+/datum/controller/subsystem/logging/Initialize()
+	. = ..()
+	for(var/datum/log_entry/entry_type as anything in subtypesof(/datum/log_entry))
+		var/entry_category = initial(entry_type.category)
+		if(!entries[entry_category])
+			entries[entry_category] = list()
+
+/datum/controller/subsystem/logging/proc/append_entry(datum/log_entry/entry)
+	LAZYADDASSOC(entries, entry.category, entry)
+
+/client/verb/view_logs()
+	SSlogging.view_logs()
```

Comment:
Might be better as an admin verb, because wouldn't this expose active in-round logs to anyone who runs this verb?

Reply:
this is a debug verb